install/kubernetes/helm/istio/charts/kiali/values.yaml

```diff
@@ -5,7 +5,7 @@ enabled: false # Note that if using the demo or demo-auth yaml when installing v
 replicaCount: 1
 hub: quay.io/kiali
 image: kiali
-tag: v1.1.0
+tag: v1.4.0
```

Comment:
BTW: this is tagging a z-stream image. In past releases, we have been tagging at the X.Y level so if Kiali needs to release a patch fix, Istio users will be able to pick up the latest z-stream patch release (because quay.io/kiali tags will point the vX.Y to the latest vX.Y.Z image).

For example, we are going to see if releasing a v1.4.1 will help Lin's issue of the Firefox renderingn problem. But that would require another change to move this istio helm values.yaml file to move to 1.4.1.

I would recommend changing this to:

```
tag: v1.4
```

To allow for Istio users to pull in patch/bug fix releases without requiring a change to the tag, unless there is a policy within Istio project to always pull in specific z-stream images.